src/components/directory-tree-view.js

```diff
@@ -153,7 +153,6 @@ const DirectoryTreeView = ({ treeViewUID, mapData }) => {
 
     const handleOpenMenu = (event) => {
         setAnchorEl(event.currentTarget);
-        event.stopPropagation();
```

Comment:
I don't see any differences then I imagine it's a good pratice to remove code which we don't understand behavior but maybe you could explain here why `event.stopPropagation()` and `e.preventDefault()` are useless